backend/apis/clients/get-client.api.js

```diff
@@ -188,3 +188,164 @@ function getClientById(clientId, cbk, connection) {
     cbk(err, client);
   });
 }
+
+exports.getAllClientsById = getAllClientsById;
+
+function getAllClientsById(clients, cbk, connection) {
+  clients = clients.map((item) => Number(item));
+
+  const getClient = mysql.format(
+    `
+    SELECT
```

Comment:
Inlining the SQL into a string is the most common pattern in this project, but I now have switched to creating `.sql` files so we can at least get some syntax highlighting and a bit of autocomplete. Let's switch to that:

For example:

https://github.com/JustUtahCoders/comunidades-unidas-internal/blob/master/backend/apis/invoices/get-client-payments.sql

https://github.com/JustUtahCoders/comunidades-unidas-internal/blob/314ef28fc185ea052c1ffe5e16b60607fac4a9ed/backend/apis/invoices/get-client-payments.api.js#L13-L15